web/webkit/src/main/scala/net/liftweb/http/Templates.scala

```diff
@@ -195,23 +183,21 @@ object Templates {
           case _ =>
             val pls = places.mkString("/", "/", "")
 
-            val se = suffixes.iterator
+            val se = parsers.iterator
             val sl = List("_" + locale.toString, "_" + locale.getLanguage, "")
 
             var found = false
             var ret: NodeSeq = null
 
             while (!found && se.hasNext) {
-              val s = se.next
+              val (suffix, parser) = se.next
               val le = sl.iterator
```

Comment:
Oh dear. I'm going to have to take aim with my variable-name-o-smasher one of these days to fix this region and make it comprehensible.